src/Illuminate/Database/Eloquent/Factory.php

```diff
@@ -116,11 +116,12 @@ public function state($class, $state, $attributes)
      *
      * @param  string  $class
      * @param  callable  $callback
+     * @param  string  $name
      * @return $this
      */
-    public function afterMaking($class, $callback)
+    public function afterMaking($class, callable $callback, $name = 'default')
```

Comment:
Please remove this extra type from the definition. It's a breaking change to the signature.